src/components/courses/CourseDetails.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="bg-white">
+        <div class="mx-auto max-w-2xl py-3 px-4 sm:py-5 sm:px-6 lg:grid lg:max-w-7xl lg:grid-cols-2 lg:gap-x-8 lg:px-8">
+            <!-- Product details -->
+            <div class="lg:max-w-lg lg:self-end">
+                <div class="mt-4">
+                    <h1 class="text-3xl font-bold tracking-tight text-gray-900 sm:text-4xl" id="title">{{ courseStore.course.title }}</h1>
+                </div>
```

Comment:
This looks like an extra div. Properties from `<div class="mt-4">` attribute can be moved to h1 class